releases/release-8.3.0.md

```diff
@@ -0,0 +1,436 @@
+
+    In v8.0.0, query performance is optimized for the following four system tables:
+
+    - `INFORMATION_SCHEMA.TABLES`
+    - `INFORMATION_SCHEMA.STATISTICS`
+    - `INFORMATION_SCHEMA.KEY_COLUMN_USAGE`
+    - `INFORMATION_SCHEMA.REFERENTIAL_CONSTRAINTS`
+
+  In v8.3.0, the query performance is optimized for the following system tables, bringing a multi-fold performance improvement compared to v8.2.0.
+
+    - `INFORMATION_SCHEMA.CHECK_CONSTRAINTS`
+    - `INFORMATION_SCHEMA.COLUMNS`
+    - `INFORMATION_SCHEMA.PARTITIONS`
+    - `INFORMATION_SCHEMA.SCHEMATA`
+    - `INFORMATION_SCHEMA.SEQUENCES`
+    - `INFORMATION_SCHEMA.TABLE_CONSTRAINTS`
+    - `INFORMATION_SCHEMA.TIDB_CHECK_CONSTRAINTS`
+    - `INFORMATION_SCHEMA.TiDB_INDEXES`
+    - `INFORMATION_SCHEMA.TIDB_INDEX_USAGE`
+    - `INFORMATION_SCHEMA.VIEWS`
+
+* Support partition pruning when partition expressions use the `EXTRACT(YEAR_MONTH...)` function to improve query performance [#54209](https://github.com/pingcap/tidb/pull/54209) @[mjonss](https://github.com/mjonss)
+
+    In previous versions, when partition expressions use the `EXTRACT(YEAR_MONTH...)` function, partition pruning is not supported, resulting in poor query performance. Starting from v8.3.0, partition pruning is supported when partition expressions use the `EXTRACT(YEAR_MONTH...)` function, which improves query performance.
+
+    For more information, see [documentation](/partition-pruning.md#scenario-three).
+
+* Improve the performance of `CREATE TABLE` by 1.4 times, `CREATE DATABASE` by 2.1 times, and `ADD COLUMN` by 2 times [#54436](https://github.com/pingcap/tidb/issues/54436) @[D3Hunter](https://github.com/D3Hunter)
+
+    TiDB v8.0.0 introduces the system variable [`tidb_enable_fast_create_table`](/system-variables.md#tidb_enable_fast_create_table-new-in-v800) to improve table creation performance in batch table creation scenarios. In v8.3.0, when submitting the DDL statements for table creation concurrently through 10 sessions in a single database, the performance is improved by 1.4 times compared with v8.2.0.
+
+    In v8.3.0, the performance of general DDLs in batch execution has improved compared to v8.2.0. The performance of `CREATE DATABASE` for 10 sessions concurrently improves by 19 times compared with v8.1.0 and 2.1 times compared with v8.2.0. The performance of using 10 sessions to add columns (`ADD COLUMN`) to multiple tables in the same database in batch has improved by 10 times compared with v8.1.0, and 2.1 times compared with v8.2.0. The performance of `ADD COLUMN` with 10 sessions on multiple tables in the same database has improved by 10 times compared with v8.1.0 and 2 times compared with v8.2.0.
+
+    For more information, see [documentation](/system-variables.md#tidb_enable_fast_create_table-new-in-v800).
+
+* Partitioned tables support global indexes (experimental) [#45133](https://github.com/pingcap/tidb/issues/45133) @[mjonss](https://github.com/mjonss)
```

Comment:
Also @Defined2014 should be attributed, as well as @jiyfhust and @L-maple (two external contributors).
Previously there have been even more contributors, like @tiancaiamao , @bb7133, @crazycs520 and more :)
Since the mentioned enhancement Issue is just a TODO :)